src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -2414,6 +2414,7 @@ public async Task ValidateMultipleCreateAndCreateMutationWhenMultipleCreateOpera
         /// </summary>
         [TestMethod]
         [TestCategory(TestCategory.MSSQL)]
+        [Ignore]
```

Comment:
Initially ignored because the error message expected will be different when request fails in request validation vs query gen.

Now that request validation is merged, updated right error message, and  unignored now.